software/lazydogpile.livecodescript

```diff
@@ -0,0 +1,954 @@
+﻿script "lazydogpile"
+/*
+This stack is mostly an empty "hull" that ties everything else together
+Some stuff, like the Livecode engine and objects, need to be binary
+Everything else is in text files
+*/
+local sMain -- main[doc][rec][tag]=data/pointers
+-- logic
+local sPointerTags -- space delimited list of pointer categories
+local sMainShort
```

Comment:
Updated the vast majority of the list of s* variables at the top of lazydogpile. I think a few of them might be OBE.